package/haproxy/Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN apt-get update && apt-get install -y \
     rsyslog \
     wget \
     haproxy \
+    socat \
```

Comment:
why add socat?

Reply:
left by mistake, was thinking of using socat to connect to the haproxy socket. But "net.Dial("unix"...)" served the purpose